packages/synapse-react-client/src/lib/containers/discussion_forum/ForumThreadEditor.tsx

```diff
@@ -50,6 +65,8 @@ export const ForumThreadEditor: React.FunctionComponent<
     isLoadingTitle ||
     isLoadingReplyUpdate
 
+  const isExistingThread = !isReply && initialTitle
```

Comment:
Move expression into a constant, so it can be reused for setting the dialog title